.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-@coralogix/team-cs-devops
\ No newline at end of file
+/.github/CODEOWNERS @coralogix/security-review
```

Comment:
`/` should be removed as far as I know, otherwise not work. Also, we need to understand who are the people behind `security-review`

Reply:
Your'e the only member of this group